### PR TITLE
Add frame widget.

### PR DESCRIPTION
**Pull request by jfmyers**

### CODE REVIEW

File: src/frame/index.tsx [hunk 1]
```diff
@@ -1,28 +1,125 @@
...
+  messageKey?: string;
+  messagePayloadType?: string;
+  messagePayloadActionKey?: string;
 };
 
 type State = {};
 
+const encodeURLParams = (url: string, data: any): string => {
+  let params = "";
+  for (const key in data) {
```
Comment (bcinman) [inline src/frame/index.tsx]: There's an easier way to encode an object with:
```js
new URLSearchParams(data).toString()
```

File: src/frame/index.tsx [hunk 2]
```diff
@@ -1,28 +1,125 @@
...
+
+    if (message.type !== messagePayloadType) return;
+
+    const payload = message[messagePayloadType];
+    if (!payload) return;
+
+    const messageAction = payload[messagePayloadActionKey];
+    if (!messageAction || typeof messageAction != "string") return;
+
+    window.location.assign(messageAction);
```
Comment (bcinman) [inline src/frame/index.tsx:52]: We might want this to open in an new tab instead?
```js
window.open(messageAction, '_blank');
```

Comment (jfmyers) [reply src/frame/index.tsx:52]: Right now the functionality is to direct the user straight to the link so I wanted to maintain that.

File: src/frame/index.tsx (hunk 1 above)
Comment (jfmyers) [reply src/frame/index.tsx]: Good call. Are there any browser compatibility issues?

Comment (bcinman) [reply src/frame/index.tsx]: Browser support is pretty solid https://caniuse.com/#search=URLSearchParams

Comment (jfmyers) [reply src/frame/index.tsx]: 👍

**Pull request merged**